server/novaboot-shell

```diff
@@ -89,6 +89,38 @@ read_config() {
     . "${NOVABOOT_SHELL_CONFIG:-$HOME/.novaboot-shell}"
 }
 
+get_nfsroot() {
+    run_unfsd
+
+    local nfsroot_path="$HOME/tftproot/$NB_USER/root"
+
+    # Use the first ip address returned by the `hostname -I` command
+    # TODO: change to specific ip or use better method
+    local nfsroot_ip=`hostname -I | cut -d' ' -f1`
+
+    local port_file="$HOME/.unfsd/$NB_USER.ports"
+
+    local OLD_IFS=$IFS
+    IFS="
+="
+
+    local nfs_tcp
+    local mount_tcp
+
+    while read -r service port
+    do
+        if [ "$service" = "NFS_TCP" ]; then
+            nfs_tcp=$port
+        elif [ "$service" = "MOUNT_TCP" ]; then
+            mount_tcp=$port
+        fi
+    done < $port_file
+
+    IFS=$OLD_IFS
```

Comment:
can be replaced with `. $port_file`, `NFS_TCP` and `MOUNT_TCP` then will be variables containing the port number